catalogs/catalog-lakehouse-iceberg/src/main/java/com/datastrato/gravitino/catalog/lakehouse/iceberg/ops/IcebergTableOps.java

```diff
@@ -117,7 +121,23 @@ public void purgeTable(TableIdentifier tableIdentifier) {
   }
 
   public LoadTableResponse loadTable(TableIdentifier tableIdentifier) {
-    return CatalogHandlers.loadTable(catalog, tableIdentifier);
+    try {
+      return CatalogHandlers.loadTable(catalog, tableIdentifier);
+    } catch (NoSuchTableException e) {
+      Table table = catalog.loadTable(tableIdentifier);
```

Comment:
could we justify whether it's a metadata table? If not a metadata table, no need to load again

Reply:
ok.

Reply:
updated.